app-text/cherrytree/cherrytree-0.99.52.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit cmake python-any-r1 xdg
+
+DESCRIPTION="A hierarchical note taking application (C++ version)"
+HOMEPAGE="https://www.giuspen.com/cherrytree/"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/giuspen/cherrytree"
+	S="${WORKDIR}/${P}/future"
+else
+	SRC_URI="https://github.com/giuspen/cherrytree/releases/download/${PV}/${P/-/_}.tar.xz"
+	S="${WORKDIR}"/${P/-/_}
+
+	KEYWORDS="~amd64 ~x86"
+	# S="${WORKDIR}/${PN}-${COMMIT}/future"
+fi
+
+# GPL-3 — future/src/ct (CherryTree)
+# LGPL-2.1 — future/src/7za (7zip)
+# MIT — future/src/fmt (libfmt)
+LICENSE="GPL-3 LGPL-2.1 MIT"
+SLOT="0"
+IUSE="nls test"
+
+# Has deps that aren't available in ::gentoo repo
+RESTRICT="test"
+
+RDEPEND="app-i18n/uchardet
+	app-text/gspell:=
+	>=dev-cpp/glibmm-2.64.2:2
+	dev-cpp/gtkmm:3.0
+	dev-cpp/gtksourceviewmm:3.0
+	dev-cpp/libxmlpp:2.6
+	dev-cpp/pangomm:1.4
+	dev-db/sqlite:3
+	dev-libs/glib:2
+	dev-libs/libfmt:=
+	dev-libs/fribidi
+	dev-libs/spdlog
+	net-misc/curl
+	x11-libs/cairo
+	x11-libs/gtk+:3
+	x11-libs/pango
+	x11-libs/vte:2.91"
+DEPEND="${PYTHON_DEPS}
+${RDEPEND}
+"
+BDEPEND="
+	virtual/pkgconfig
+	nls? (
+		sys-devel/gettext
+	)
+	test? ( dev-util/cpputest )"
+
+src_prepare() {
+	# disable compress man pages
+	sed -i -e \
+		'/install(FILES/s|${MANFILE_FULL_GZ}|${CMAKE_SOURCE_DIR}/data/cherrytree.1|' \
+		CMakeLists.txt || die
+
+	if [[ ${PV} != *9999 ]]; then
+		sed -i \
+			-e "/^set(CT_VERSION/s|\"\(.*\)\"|\"${PV}\"|" \
+			CMakeLists.txt || die
+	fi
```

Comment:
It doesn't look like this is doing anything, could you check if this is still relevant?
```
 * SED: the following did not cause any changes
 *         sed -i -e "/^set(CT_VERSION/s|\"\(.*\)\"|\"${PV}\"|" CMakeLists.txt || die;
 * no-op: -e /^set(CT_VERSION/s|"\(.*\)"|"0.99.52"|
```

Reply:
references to dev version 9999 will be removed on next commit